static-exporter/main.libsonnet

```
@@ -46,6 +48,18 @@ local k = import 'ksonnet-util/kausal.libsonnet';
         ),
     }),
 
+  withHttpConfig():: {
+    local configMap = k.core.v1.configMap,
+    local volumeMount = k.core.v1.volumeMount,
+    httpdConfig:
+      configMap.new(self.name, 'httpd-config')
+      + configMap.withData({
+        'httpd.conf': importstr 'httpd.conf',
```

```suggestion
  withHttpConfig(config=(importstr 'httpd.conf')):: {
    local configMap = k.core.v1.configMap,
    local volumeMount = k.core.v1.volumeMount,
    httpdConfig:
      configMap.new(self.name, 'httpd-config')
      + configMap.withData({
        'httpd.conf': config,
```
I'd set the default in the argument, that'll allow downstream to override it with their own.